modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -62,6 +62,11 @@ spec:
       statsPort: 1936
 ----
 ====
++
+[NOTE]
+====
+On {rh-openstack-first} platform `LoadBalancerService` `endpointPublishingStrategy` is only supported when cloud provider is configured to create health monitors. Please note that as of {rh-openstack} 16.1 and 16.2 this is only possible with Amphora Octavia provider.
```

Comment:
Can you help me understand how can I link to the cloud provider configuration document from here?

Reply:
@maxwelldb: Exactly the other one I edited, which explains how to configure the cloud provider with health monitors and why it's important.

Reply:
Oh, that's just a module. So I guess that module on the regular OpenStack installation page will work, so like to [1].

[1] https://docs.openshift.com/container-platform/4.10/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-setting-cloud-provider-options_installing-openstack-installer-custom

Reply:
An unfortunate limitation of our docs tooling is that you [cannot link from one module to another within the same docs set](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references). While we know that the installation pages are pretty reliable, it isn't 'guaranteed' that a module will ever be packaged with another, and so the practice is forbidden. 😬 

We can get around this by either quoting the title of the section we mean--it's on the user to locate it, then--or by adding a note to the assembly that contains the module that we want to link from. Totally a judgment call, though we don't really like keeping too much content in assemblies when possible.